control-plane/pkg/security/secret.go

```diff
@@ -175,18 +175,24 @@ func sslConfig(protocol string, data map[string][]byte) kafka.ConfigOption {
 			if !skipClientAuth {
 				userKeyCert, ok := data[UserKey]
 				if !ok {
-					return fmt.Errorf(
-						`[protocol %s] required user key (key: %s) - use "%s: true" to disable client auth`,
-						protocol, UserKey, UserSkip,
-					)
+					userKeyCert, ok = data["tls.key"]
+					if !ok {
+						return fmt.Errorf(
+							`[protocol %s] required user key (key: %s) - use "%s: true" to disable client auth`,
+							protocol, UserKey, UserSkip,
+						)
+					}
 				}
 
 				userCert, ok := data[UserCertificate]
 				if !ok {
-					return fmt.Errorf(
-						`[protocol %s] required user key (key: %s) - use "%s: true" to disable client auth`,
-						protocol, UserCertificate, UserSkip,
-					)
+					userCert, ok = data["tls.crt"]
+					if !ok {
+						return fmt.Errorf(
+							`[protocol %s] required user key (key: %s) - use "%s: true" to disable client auth`,
+							protocol, UserCertificate, UserSkip,
+						)
+					}
```

Comment:
There is a bit of repetition here: maybe we could extract a function like
```golang
func oneValueOf(data map[string][]byte, keys...string) ([]byte, bool) {
   // TODO go through keys and find the first present and non empty key in data
}
```

and then we can use it like this:
```golang
userKeyCert, ok := oneValueOf(data, UserKey, "tls.key")
/// ...
userCert, ok := oneValueOf(data, UserCertificate, "tls.crt")
```

Reply:
Done 👍🏽